.github/workflows/test-env-deploy.yml

```
@@ -68,11 +68,21 @@ jobs:
           pattern: (http|https)://[a-zA-Z0-9.-]+/?
         run: |
           echo "::set-output name=custom_marketplace_url::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
+      - name: Get APPS_MARKETPLACE_API_URI
+        id: apps_marketplace_api_uri
+        # Search for APPS_MARKETPLACE_API_URI in PR description
+        env:
+          pull_request_body: ${{ github.event.pull_request.body }}
+          prefix: APPS_MARKETPLACE_API_URI=
+          pattern: (http|https)://[a-zA-Z0-9.-]+[\w/-]+/?
+        run: |
+          echo "::set-output name=custom_apps_marketplace_api_uri::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
```

Quote the untrusted var

```suggestion
          echo "::set-output name=custom_apps_marketplace_api_uri::$(echo "$pull_request_body" | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
```